cmd/webhook/main.go

```diff
@@ -14,7 +14,41 @@
 
 package main
 
-func main() {
+import (
```

Comment:
I must be going crazy because I think I already reviewed this code :)

Did you rebase your branch before submitting the PR?
```
git fetch --all
git checkout init
git rebase upstream/main
git push -f origin init
```

Reply:
My bad - forgot to fetch upstream earlier, should be fixed now